bpo-35148: fix resetting codepage in activate.bat

moltob opened the pull request:
The current version of the Windows `activate.bat` activation script unfortunately does not work well in some international versions of Windows.

E.g. on a German Windows 10 version (or the bug reporter's Swiss version) the output of `chcp` is something like this:

```
Aktive Codepage: 850.
```

Unlike the US version, the statement ends with a period, unexpected by the activation script, leading to an unsuccessful attempt to reset the codepage after venv installation. While the venv is set up correctly, the codepage is not reset. In addition a user-visible error message appears, confusing users and giving the doubt whether venv actually worked:

```
Parameterformat falsch - 850.
```

 (English: "wrong parameter format")

The fix leaves the original approach in place, but deletes any periods from the CP number string, if they exist, which means it simply cleans up the CP string. This should have minimal side effects.


<!-- issue-number: [bpo-35148](https://bugs.python.org/issue35148) -->
https://bugs.python.org/issue35148
<!-- /issue-number -->

moltob committed:
fix parsing chcp output

moltob committed:
added news

eryksun commented on Lib/venv/scripts/nt/activate.bat:
```diff
@@ -1,7 +1,7 @@
 
 rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
+for /f "tokens=2 delims=:." %%a in ('"%SystemRoot%\System32\chcp.com"') do (
```
I suggested this on PR 10295, but there were concerns that it wasn't a robust solution.

moltob committed:
alternative fix

This leaves the original approach untouched and simply removes a period, if it is an unwanted part of the CP string.

moltob committed:
less intrusive
